src/tm-threads.c

```diff
@@ -2199,14 +2199,24 @@ void TmThreadsSetThreadTimestamp(const int id, const SCTime_t ts)
     int idx = id - 1;
     Thread *t = &thread_store.threads[idx];
     t->pktts = ts;
-    struct timeval systs;
-    gettimeofday(&systs, NULL);
-    t->sys_sec_stamp = (uint32_t)systs.tv_sec;
+    struct timeval tv;
+    gettimeofday(&tv, NULL);
+    SCTime_t now = SCTIME_FROM_TIMEVAL(&tv);
+
+    if (t->sys_sec_stamp.secs != 0) {
+        SCTime_t tmpts = SCTIME_ADD_SECS(t->sys_sec_stamp, 3);
```

Comment:
Is there a reason for using ~4 seconds == long time?

Reply:
it's ore a left over of some debugging where I wanted to know the effect of inactive threads on the timestamp the FM used. Turned it into debug.